dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.globalhandler-1.0.feature

```diff
@@ -3,8 +3,12 @@ symbolicName=com.ibm.websphere.appserver.globalhandler-1.0
 WLP-DisableAllFeatures-OnConflict: false
 visibility=private
 IBM-SPI-Package: com.ibm.wsspi.webservices.handler
+-features=\
+  io.openliberty.servlet.api-3.0; ibm.tolerates:="3.1,4.0,5.0"; apiJar=false, \
+  io.openliberty.globalhandler1.0.internal.ee-6.0; ibm.tolerates:="9.0"
 -bundles=com.ibm.ws.webservices.handler
--jars=com.ibm.websphere.appserver.spi.globalhandler; location:=dev/spi/ibm/
+-jars=com.ibm.websphere.appserver.spi.globalhandler; location:=dev/spi/ibm/, \
+  com.ibm.websphere.appserver.spi.globalhandler.jakarta; location:=dev/spi/ibm/
```

Comment:
IIUC, this will package the transformed and untransformed versions in all Jakarta/JavaEE flavors of Open Liberty - increasing the install footprint (and adding dead classes). Are we ok with that?

Reply:
I am ok with it. We could make some updates to the feature tooling to not make this necessary, but right now it doesn't look at the tolerated private features to find -jar files that would be needed for a particular private feature, so right now this is the only way for it to think that jar should be included.  In general I think it is an ok thing because those jars are relevant for the feature itself.  It depends on the other features that are used to determine which one to compile against and use.  I will be making more changes like this in the coming weeks due to the limitations of the feature tooling in liberty right now.  Even if the feature tooling did it the right way, it would end up not knowing which of the tolerate was going to be done so it would pick both anyway so in the end it is the same result.